alns/accept/tests/test_adaptive_threshold.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from numpy.testing import assert_, assert_equal, assert_raises
+from pytest import mark
+
+from alns.accept.AdaptiveThreshold import AdaptiveThreshold
+
+
+class MockCandidate:
```

Comment:
You can get a mock like this from `alns.tests.states`, where it is called `VarObj`. There is no need to redefine one here.